lib/metanorma/ogc/processor.rb

```diff
@@ -25,8 +25,8 @@ def fonts_manifest
           "Arial" => nil,
```

Comment:
@Intelligent2013 Lato also has other subfamilies, like Lato Light. Any chance we need them?

Reply:
Let's add Lato Light.

Reply:
Thanks and done :)